src/oci.h

```diff
@@ -137,6 +137,11 @@
 /* Path to the stateless passwd file. */ 
 #define STATELESS_PASSWD_PATH "/usr/share/defaults/etc/passwd"
 
+/* Offset to add to the interface index for assigning the pci slot.
+ * First 3 slots are in use for pc-lite machine type
+ */
+#define PCI_OFFSET 4
```

Comment:
Is there any likelihood that qemu might change the number of slots it assigns itself? What are the first 3 slots used for specifically?

This looks like a potential maintenance issue, but I'm not sure how we can protect against future changes here? Hopefully, if qemu did change, we'd get errors logged in the hypervisor logs, but can we do better than that?